### PR TITLE
Alex Fernandez talk og & main

### PR DESCRIPTION
**Pull request by jluis**
The talk is accepted but it still lacks confirmation. 
onion OG log on repo.
talk id code-you-future

### CODE REVIEW

**Comment by meis** [comment]
Thanks for the work!

I like the fa-icon but I'm a bit concerned that it's a logo of a different nonprofit organization. GO ahead if you don't think that'll be a problem.

**Comment by jluis** [comment]
I was doubting between the code camp and the code fork

**Comment by meis** [comment]
And `fa-code`?

File: 2017/index.html [hunk 1]
```diff
@@ -279,6 +279,16 @@
...
                 </li>
+                 <li class="icon fa-code-fork" id="/talks/code-you-future">
+                  <h3>Code Your Future - "Escuela de programación para refugiados"</h3>
+                  <h4>Alex Fernadez</h4>
+                  <p>
+                    CodeYourFuture ubicada en Gran Bretaña es la primera, y la única, escuela de programación que tiene como unica finalidad la de enseñar a los refugiados a programar. La mitad de los estudiantes vienen de Sira, otros vienen de Uganda, Etiopía, Afganistán y uno que es oficialmente reconocido como sin estado por el gobierno de Gran Bretaña. Ninguno puede volver a su hogar ni a sus vidas anteriores y aprenden a programar con el fin de encontrar un trabajo estable en Gran Bretaña. Esperamos que después de seis meses, cada estudiante tenga los suficientes conocimientos de programación para encontrar un trabajo como desarrollador web.
+                  </p>
+                  <p>
+                    Do you want to help us set up a community in Barcelona?
```
Comment (meis) [inline 2017/index.html]: We translated everything to spanish but this fragment, I suppose we want to also do it.

Comment (jluis) [reply 2017/index.html]: Ok .
I didn't like how it sounds in spanish as a questinon :
  ¿Quieres ayudarnos a crear una comunidad en Barcelona?
   o
¿Ayudarias a crear una comunidad en Barcelona?

Comment (meis) [reply 2017/index.html]: I like the first one more: `¿Quieres ayudarnos a crear una comunidad en Barcelona?`

**Pull request merged**